cpp/modmesh/view/RManager.cpp

```diff
@@ -217,50 +206,54 @@ void RManager::setUpMenu()
         use_fps_camera->setCheckable(true);
         use_orbit_camera->setChecked(true);
 
-        m_cameraMenu->addAction(use_orbit_camera);
-        m_cameraMenu->addAction(use_fps_camera);
+        m_viewMenu->addAction(use_orbit_camera);
+        m_viewMenu->addAction(use_fps_camera);
     }
 
-    {
-        m_appMenu = m_mainWindow->menuBar()->addMenu(QString("App"));
-
-        this->addApplication(QString("sample_mesh"));
-        this->addApplication(QString("linear_wave"));
-        this->addApplication(QString("bad_euler1d"));
-    }
-
-    {
-        QAction * params = new RAction(
-            QString("Parameters"),
-            QString("Runtime parameters"),
-            []()
-            {
-                static int64_t int64V = 5566;
-                static double doubleV = 77.88;
-                auto params = createParameters();
-                addParam(params, "global.a.b.int64_foo", &int64V);
-                addParam(params, "global.a.b.double_bar", &doubleV);
-                openParameterView(params);
-            });
-        m_fileMenu->addAction(params);
-    }
+    m_oneMenu = m_mainWindow->menuBar()->addMenu(QString("One"));
+    m_oneMenu->addAction(new RAppAction(
```

Comment:
After simplifying the code to add the menu items, it is not necessary to use a local scope because no temporary variable is used.